docker-compose.yml

```diff
@@ -0,0 +1 @@
+build/dev/docker-compose.yml
```

Comment:
Running `docker-compose up -d` from the repository root does not work for me, because the symbolic link messes up the relative paths used in `build/dev/docker-compose.yml`.

Example: Docker wants to mount `$HOME/example/nginx.conf` instead of `$HOME/agdsn/sipa/example/nginx.conf` into the nginx container.

Reply:
Yes, this is a known flaw and the reason for such a symlink not existing atm.

Reply:
```suggestion
```